docs/admin/install/install.rst

```diff
@@ -152,13 +159,13 @@ With the key and configuration available in ``dom0``, you're ready to set up Sec
 
```

Comment:
The f32 in the `[securedrop-workstation-temporary]` section needs to be switched to f37 as well.